remote-robot/src/main/kotlin/com/intellij/remoterobot/RemoteRobot.kt

```diff
@@ -63,6 +66,23 @@ class RemoteRobot @JvmOverloads constructor(
     val os: String
         get() = callJs("com.intellij.openapi.util.SystemInfo.OS_NAME")
 
+    /**
+     * Get shot of whole screen in '.png' format.
+     * Use ImageIO.write method with "png" formatName
+     */
+    fun getScreenshot(): BufferedImage {
+        val bytes = ideRobotClient.makeScreenshot()
+        return ImageIO.read(ByteArrayInputStream(bytes))
+    }
+
+    /**
+     * Get shot of component in '.png' format.
+     * Use ImageIO.write method with "png" formatName
+     */
+    fun getScreenshot(componentId: String): BufferedImage {
```

Comment:
I think this method is not as useful here as it could be in `Fixture` class. And we also already have `componentId` there

Reply:
fixed